corehq/apps/cloudcare/templates/cloudcare/partials/grid_view/landing_page_app.html

```diff
@@ -14,5 +14,4 @@ <h3><%- appName %></h3>
       </div>
     </div>
   </div>
-  </div>
```

Comment:
Good find. How did you see that?

Reply:
Good indentation style! It looked weird so I went back and checked it was a real error not an indentation error.